docs/source/conf.py

```diff
@@ -27,13 +27,14 @@
 sys.path.insert(0, os.path.abspath(PATH_ROOT))
 sys.path.append(os.path.join(PATH_RAW_NB, ".actions"))
 
-_SHOULD_COPY_NOTEBOOKS = True
+_SHOULD_COPY_NOTEBOOKS = bool(int(os.getenv("SHOULD_COPY_NOTEBOOKS", 1)))
```

Comment:
I'm in favour of adding `PL` prefix to the env var just in case it's used by other programs.
```suggestion
_SHOULD_COPY_NOTEBOOKS = bool(int(os.getenv("PL_SHOULD_COPY_NOTEBOOKS", 1)))
```

Reply:
Done!